### DISCUSSION TITLE
getActiveAppInfo always return 'kit' ? 🧐

### DISCUSSION THREAD

**Discussion by alkene5** [johnlindquist/kit#1355] [Errors] [answered]
<img width="1054" alt="image" src="https://github.com/johnlindquist/kit/assets/61105068/cd44ccf5-eda3-4cca-a1c4-4c1254d69337">

**Comment by johnlindquist** [comment, marked as answer]
This should be fixed in an upcoming build. I'm building out a custom window type for Script Kit to have better control over focus/frontmost/active/spaces/etc.
